coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/person/ProposalPersonnelPage.xml

```diff
@@ -54,7 +54,7 @@
           p:actionParameters="PropDev-PersonnelPage-Wizard.step:0" p:iconClass="icon-user"/>
 
     <bean id="PropDev-PersonnelPage-Wizard" parent="Kc-Wizard"
-          p:fieldBindingObjectPath="addKeyPersonHelper">
+          p:fieldBindingObjectPath="addKeyPersonHelper" p:retrieveViaAjax="true" >
```

Comment:
retrieveViaAjax is not required here for the wizard to work with key personnel. Removing this from here. Tested it using the Proposal Key Personnel Page.